frontend/src/components/GoalsTable/GoalRow.css

```diff
@@ -1,3 +1,57 @@
+.goals-table .usa-table .tta-smarthub--goal-row td:first-child {
+  border-left: 1px solid #c5c5c5;
+  min-width: 160px;
+}
+
+.goals-table .usa-table .tta-smarthub--goal-row td:last-child {
+  border-right: 1px solid #c5c5c5;
+}
+
+.goals-table .usa-table .tta-smarthub--goal-row td:nth-child(2) {
+  min-width: 120px;
+}
+
+.goals-table .usa-table .tta-smarthub--goal-row td:nth-child(3) {
+  max-width: 500px;
+  min-width: 500px;
+}
+
+.goals-table .usa-table .tta-smarthub--goal-row td:nth-child(4) {
+  max-width: 330px;
+  min-width: 330px;
+}
+.goals-table .usa-table .tta-smarthub--goal-row td:nth-child(5) {
+  min-width: 168px;
+}
+
+.goals-table .usa-table .tta-smarthub--goal-row-objectives-enabled {
+  cursor: pointer;
+}
+
+.goals-table .usa-table .tta-smarthub--goal-row-objectives-disabled {
```

Comment:
What's pointer-events: none for?

Reply:
If there are no objectives.

Reply:
Oh ok. We show the button if there are 0 objectives?